src/config.json

```
@@ -728,6 +728,7 @@
     "cactus.chat"
   ],
   "blacklist": [
+    "metamasksupport.com',
```

Accidental single-quote
```suggestion
    "metamasksupport.com",
```